dekkProjekt/resources/views/map_leaflet.blade.php

```diff
@@ -1,93 +1,221 @@
+
+                    <script>
+                    var slider = document.getElementById("myRange");
+                    var output = document.getElementById("sliderYear");
+                    output.innerHTML = slider.value;
+                    slider.oninput = function() {
+                        output.innerHTML = this.value;
+                    }
+                    </script>
+                </div>
+
+                <!-- Send requested datasets -->
+                <button type="button" class="btn btn-dark" onclick="sendRequest()">Obnoviť</button>
+            </div>
+
+            <!-- Map -->
+            <div class="col-xl-5 col-lg-12">
+                <div id='map'></div>
+                <div class="row">
+
+                    <div class="col-xl-6 col-lg-6">
+                        <div class="picker1" id="picker1">
+                            <script>
+                            var colorPicker1 = new iro.ColorPicker('#picker1', {
+                                width: 150,
+                                color: "#fff",
+                                layout: [{
+                                    component: iro.ui.Wheel,
+                                    options: {}
+                                }, ]
+                            });
+                            </script>
+                        </div>
+                    </div>
+
+                    <div class="col-xl-6 col-lg-6">
+                        <div class="picker2" id="picker2">
+                            <script>
+                            var colorPicker2 = new iro.ColorPicker('#picker2', {
+                                width: 150,
+                                color: "#fff",
+                                layout: [{
+                                    component: iro.ui.Wheel,
+                                    options: {}
+                                }, ]
+                            });
+                            </script>
+                        </div>
+                    </div>
+                </div>
+            </div>
+
+            <!-- BEFORE : <div class="col-md-4" id="odpoved"></div> -->
+            <div class="col-xl-4 col-lg-12" id="sidePanel">
+                <!-- Graph -->
+                <div id="my_dataviz3">
+                    <h3>Graf korelácie</h3>
+                </div>
+                <div class="correlation_meaning">
+                    <p>Korelačný koeficient je <a class="red"
+                            href="https://sk.wikipedia.org/wiki/Korelácia_(štatistika)"> 0.64 </a> a hovorí nám...</p>
+                </div>
+                <!-- Graph save button -->
+                <button id='saveButtonGraph' type="button" class="btn btn-dark">Stiahnuť PNG grafu</button>
+                <!-- Map save button -->
+                <button id='saveButtonMap' type="button" class="btn btn-dark">Stiahnuť PNG mapy</button>
+            </div>
+        </div>

```

Comment:
prehodila by som tieto tlacitka nech to na stiahnutie mapy je blizsie k mape

Reply:
neviem, ja by som ich nechal obe v pravom panely ale v návrhu máme, že pre mapu bude v strednom panely s mapou, takže ho asi tam hodím, jo?

Reply:
akoze nechaj ich v pravom panely lne namiesto |stiahni graf| |stiahni mapu| daj |stihani mapu| |stiahni graf|